integration-tests/src/test/resources/features/action-sendtobridge.feature

```diff
@@ -0,0 +1,60 @@
+Feature: SendToBridge Action tests
+
+  Background:
+    Given authenticate against Manager
+    And create a new Bridge "bridge1"
+    And the Bridge "bridge1" is existing with status "ready" within 4 minutes
+    And the Ingress of Bridge "bridge1" is available within 2 minutes
+
+  Scenario: Send Cloud Event to SendToBridge with bridgeId defined
+    Given create a new Bridge "bridge2"
+    And the Bridge "bridge2" is existing with status "ready" within 4 minutes
```

Comment:
One performance comment, it may be better to create both Bridges and Processors at the same time to save some test execution time.